management/admin.py

```diff
@@ -0,0 +1,47 @@
+from discord.ext import commands
+
+
+# TODO: GM-Only checks
+class Admin:
+    """Admin-only commands that make the bot dynamic."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(hidden=True)
+    async def load(self, ctx, *, module: str):
+        """Loads a module."""
+        try:
+            self.bot.load_extension(module)
+        except Exception as e:
+            await ctx.channel.send('\N{PISTOL}')
+            await ctx.channel.send('{}: {}'.format(type(e).__name__, e))
+        else:
+            await ctx.channel.send('\N{OK HAND SIGN}')
+
+    @commands.command(hidden=True)
+    async def unload(self, ctx, *, module: str):
+        """Unloads a module."""
+        try:
+            self.bot.unload_extension(module)
+        except Exception as e:
+            await ctx.channel.send('\N{PISTOL}')
+            await ctx.channel.send('{}: {}'.format(type(e).__name__, e))
+        else:
+            await ctx.channel.send('\N{OK HAND SIGN}')
+
+    @commands.command(name='reload', hidden=True)
+    async def _reload(self, ctx, *, module: str):
+        """Reloads a module."""
+        try:
+            self.bot.unload_extension(module)
+            self.bot.load_extension(module)
+        except Exception as e:
+            await ctx.channel.send('\N{PISTOL}')
+            await ctx.channel.send('{}: {}'.format(type(e).__name__, e))
+        else:
+            await ctx.channel.send('\N{OK HAND SIGN}')
+
```

Comment:
make all that GM only please. see https://github.com/werewolves-devs/Werewolf_Bot/blob/5064b641813bc9971c541aba0ddca41f4fe6b757/utils.py#L40